### DISCUSSION TITLE
Test runner stays in test are loading React+vite

### DISCUSSION THREAD

**Discussion by alyaothman14** [cypress-io/cypress#22969] [Component Testing]
Using React +Vite and when running the spec I get the following
Using Cypress 10.3.1
Not sure what am I missing or how to debug this
Also cypress debug does not give much info about what is happening or why it hangs
<img width="1349" alt="image" src="https://user-images.githubusercontent.com/87079479/181519659-9f8c3641-cde1-4d48-bbbe-90950fb074aa.png">

**Comment by stavares843** [comment]
hello! 👋 

in case this [issue](https://github.com/cypress-io/cypress/issues/14175) helps. Seems most people reporting this issue were using the Sophos anti-virus, is that the case?
